packages/salesforcedx-vscode-soql/src/editor/queryRunner.ts

```diff
@@ -42,6 +39,7 @@ export class QueryRunner {
   private flattenQueryRecords(rawQueryRecords: JsonMap[]) {
     // filter out the attributes key
     return rawQueryRecords.map(
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
nit: do we need the eslint disable here?

Reply:
Ts lint did not like that we were ignoring the `attributes` variable that is declared but not used. I think its just a symptom of the fancy destructuring syntax, where the intention _is_ to ignore the attributes variable.
`({ attributes, ...cleanRecords }) => cleanRecords`

Reply:
You could configure a `argsIgnorePattern` for this rule to ignore args started with underscore https://eslint.org/docs/rules/no-unused-vars#argsignorepattern

Also I don't think we enforce this in other extensions.

Reply:
cool, I'll try that

Reply:
or rather I may remove the rule, the underscore option wont work for me in this case as the variable needs to match the key name

Reply:
Addressed in the `fix(soql): update eslint rules` commit